diff --git a/Makefile b/Makefile
index 05271523..1af87384 100644
--- a/Makefile
+++ b/Makefile
@@ -134,11 +134,9 @@ shared-deps:
 	$(call MAKE_CRYPTOPP, $(MAKE) shared)
 
 release-deps:
-	$(call MAKE_CRYPTOPP, $(MAKE) static)
 
 release-static-deps:
 	$(eval cmake-kovri += $(cmake-static-deps))
-	$(call MAKE_CRYPTOPP, $(MAKE) static)
 
 #-----------------------------------#
 # For local, end-user cloned builds #
